nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyGenericsTests.java

```diff
@@ -1615,6 +1615,22 @@ public void testForNullTypeRhsTypeForArrayType() {
         .doTest();
   }
 
+  @Test
+  public void testForUtilFunctionLibModel() {
+    makeHelper()
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.Nullable;",
+            "import java.util.function.Function;",
+            "class Test {",
+            "  static void testNegative() {",
+            "   Function<String,@Nullable String> removeA = a -> a.replace(\"a\",\"\");",
```

Comment:
Let's write more tests to show the type argument is being handled correctly.  E.g., check that passing in `@Nullable` when the first type argument is not `@Nullable` yields an error, etc.

Reply:
This comment is still not addressed